04_quicksort/csharp/06_GetGCD/program.cs

```diff
@@ -4,28 +4,37 @@
 
 namespace GCD
 {
-    public class program
+    public class Program
     {
+        static void Main(string[] args)
+        {
+            var lst = new List<int> { 32, 696, 40, 50 };
+            var GCD = GetGCD(640, 1680);
+            var GCDList = GetGCDList(lst);
+
+            Console.WriteLine(GCD);
+            Console.WriteLine(GCDList);
+        }
 
         //Get great Comman Divisor
-        public static int GetGCD(int FirstNumber, int SecondNumber)
-            => SecondNumber == default ? FirstNumber : GetGCD(SecondNumber, FirstNumber % SecondNumber);
+        public static int GetGCD(int firstNumber, int secondNumber)
+            => secondNumber == default ? firstNumber : GetGCD(secondNumber, firstNumber % secondNumber);
 
         //Get great Comman Divisor of list
-        public static int GetGCDList(List<int> lst)
+        public static int GetGCDList(IEnumerable<int> lst)
```

Comment:
What's the advantage of `IEnumerable` over `List`?

Reply:
All collections are implementing `IEnumerable` interface and it's used to itterate through it. I used it that you can call method with any collection or itterable object, even simple array. Preferable use `IEnumerable` over `List` when you don't need modify collection. `IEnumerable` is read-only and `List` is not.